test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -160,15 +160,50 @@ function copyManifestToNodes() {
 FLOW_AGG_YML="/tmp/flow-aggregator.yml"
 SAVED_FLOW_AGG_IMG=/tmp/flow-aggregator.tar
 FLOW_AGG_IMG_NAME=projects.registry.vmware.com/antrea/flow-aggregator:latest
+
+CH_OPERATOR_INSTALL_BUNDLE_BASE_YML=$THIS_DIR/../../../../build/yamls/clickhouse-operator-install-bundle.yml
+CH_OPERATOR_INSTALL_BUNDLE_YML="/tmp/clickhouse-operator-install-bundle.yml"
+FLOW_VIS_YML="/tmp/flow-visibility.yml"
+
+# images inside flow-visibility and clickhouse
+SAVED_GRAFANA_IMG=/tmp/flow-visibility-grafana.tar
+GRAFANA_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-grafana:8.3.3
+SAVED_CH_SERVER_IMG=/tmp/flow-visibility-clickhouse-server.tar
+CH_SERVER_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-clickhouse-server:21.11
+SAVED_CH_MONITOR_IMG=/tmp/flow-visibility-clickhouse-monitor.tar
+CH_MONITOR_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-clickhouse-monitor:latest
+SAVED_CH_OPERATOR_IMG=/tmp/flow-visibility-clickhouse-operator.tar
+CH_OPERATOR_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-clickhouse-operator:0.18.2
+SAVED_METRICS_EXPORTER_IMG=/tmp/flow-visibility-metrics-exporter.tar
+METRICS_EXPORTER_IMG_NAME=projects.registry.vmware.com/antrea/flow-visibility-metrics-exporter:0.18.2
```

Comment:
instead of doing these one by one, which introduces one more place to take care of when image versions are changed or new images are added in the manifests, should we consider getting the image name and tags directly from `clickhouse-operator-install-bundle.yml` and generated `flow-visibility.yml`? see [example](https://github.com/wsquan171/antrea/commit/cc8ffd3f5798101a41b334aea0f2585f9f498022#diff-3668450fd9c63ba472d3380fd7598085a62d867e1b0ded32573757865c742b78R448)

Reply:
I've modified it to directly get image names from flow-visibility.yml and clickhouse-operator-install-bundle.yml